tests/Object/Client/ClientPropertyTest.php

```diff
@@ -0,0 +1,80 @@
+<?php namespace ExpandOnline\KlipfolioApi\Tests\Object\Client;
+
+use ExpandOnline\KlipfolioApi\Connector\Client\ClientPropertyConnector;
+use ExpandOnline\KlipfolioApi\Connector\Klip\KlipSchemaConnector;
+use ExpandOnline\KlipfolioApi\Exception\KlipfolioApiException;
+use ExpandOnline\KlipfolioApi\Object\Client\ClientProperty;
+use ExpandOnline\KlipfolioApi\Response;
+use ExpandOnline\KlipfolioApi\Tests\Object\BaseApiResourceTest;
+use ExpandOnline\KlipfolioApi\Object\Klip\KlipSchema;
+
+/**
+ * Class KlipSchemaTest
+ * @package ExpandOnline\KlipfolioApi\Tests\Object\Klip
+ */
+class ClientPropertyTest extends BaseApiResourceTest
+{
+    
+    /**
+     * @param array $params
+     * @return KlipSchemaConnector
+     */
+    protected function getConnectorToTest($params = [])
+    {
+        return new ClientPropertyConnector($params);
+    }
+
+    /**
+     * @return KlipSchema
+     */
+    protected function getObjectToTest()
+    {
+        return new ClientProperty();
+    }
+
+    /**
+     * @return array
+     */
+    protected function getTestData() {
+        return [
+            'id' => 'test.id',
+            'properties' => [
+                'a' => '1',
+                'b' => '2'
+            ]
+        ];
+    }
+
+    public function testValidCreate() {
```

Comment:
Misschien hier hetzelfde doen als `testValidDelete()`, anders zijn we basically een andere class aan het testen.